app/src/main/java/fr/free/nrw/commons/MediaDataExtractor.java

```diff
@@ -34,23 +42,78 @@ public MediaDataExtractor(MediaWikiApi mwApi,
 
     /**
      * Simplified method to extract all details required to show media details.
-     * It fetches media object, deletion status and talk page for the filename
+     * It fetches media object, deletion status and talk page and caption for the filename
```

Comment:
-> `It fetches media object, deletion status, talk page and captions for the filename`